test/config/gatewayclass.yaml

```diff
@@ -16,6 +16,7 @@ metadata:
   name: proxy-config
   namespace: envoy-gateway-system
 spec:
+  ipFamily: IPv4
```

Comment:
IpFamily needs to be set explicitly, `sed` will change it to right value.

Reply:
Nit: use `{IP_FAMILY}` instead of `IPv4` to make it clearer?

Reply:
use Ipv4 directly allow you apply this file without run `make prepare-ip-family` for debugging